src/hooks/earn/useAccountAddress.ts

```diff
@@ -0,0 +1,16 @@
+import { useAccount } from '@lifi/wallet-management';
+import { Hex } from 'viem';
+
+const isHex = (address: string): address is Hex => {
+  return /^0x[0-9A-Fa-f]+$/.test(address);
+};
+
+export const useAccountAddress = (): Hex | undefined => {
+  const { account } = useAccount();
+
+  if (account.address && isHex(account.address)) {
```

Comment:
Can we use `https://viem.sh/docs/utilities/isHex#ishex` ?

Reply:
🤦‍♂️ nice catch!